snowflake/README.md

```diff
@@ -117,3 +141,84 @@ Snowflake credentials are required.
 
 We are in the process of adding support for applications to Snowflake-powered
 Theia, starting with NetworkPolicy recommendation.
+
+## Network flow visibility with Grafana
+
+We use Grafana as the tool to query data from Snowflake, and visualize the
+networking flows in the cluster(s).
+
+### Configure datasource
+
+Datasource and pre-built dashboards will be configured automatically in Grafana.
+To configure datasource, export the following environment variables:
+`SNOWFLAKE_WAREHOUSE`, `SNOWFLAKE_DATABASE`, `SNOWFLAKE_ROLE`(Optional).
```

Comment:
I don't think it's necessary to list them again here since they are already listed in the table. I feel like this line could be removed altogether.